napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -270,12 +451,16 @@ def _count_visible(self) -> int:
     @Slot(PackageMetadata)
     def addItem(
         self,
-        project_info: PackageMetadata,
+        project_info_versions: Tuple[PackageMetadata, list, list],
```

Comment:
I would like to see content of this lit in type annotation. Did I correctly understand it should be List[int]?

And why list, not tuple or Seqence?

Reply:
Its a List[str].  A list is what is returned when you search for the versions available in each package.  I save them in a tuple with the PackageMetadata.